src/components/button/_button.scss

```
@@ -109,17 +109,16 @@ $button-border-height: 3px;
     }
   }
 
-  &--print &__inner {
+  &--print &__inner:not([class*='icon--']) {
```

Do we still need these overrides?
```suggestion
  &--print &__inner {
```

---

Yeah don't think we need these if we aren't doing a custom icon param anymore. Ill remove them